### ISSUE TITLE
Video does not have keyboard focus, common keyboard shortcuts seem broken

### ISSUE THREAD

**Issue by scanlime** [Chocobozzz/PeerTube#2050]
**What happened?**

It looks like the player is no longer getting focus automatically. Keyboard shortcuts don't work until the player is explicitly focused, like by clicking it (which also pauses).

**What do you expect to happen instead?**

In most web video players, I expect shortcuts like "f" for fullscreen or arrow keys to seek, and I expect that they will work without any special action to focus the video player component itself, particularly if the video is autoplaying on the page.

**Steps to reproduce:**

1. Open a video

2. Video autoplays

3. Press "f", no fullscreen

**Additional information**
* PeerTube version or URL: 1.4.0-rc1
* Browser name/version: Firefox 68.0.2
* NodeJS version: 10.16.3

* Link to browser console log if useful:
* Link to server log if useful (journalctl or /var/www/peertube/storage/logs/):

**Comment by Chocobozzz** [comment]
Fixed in https://github.com/Chocobozzz/PeerTube/commit/d275e75453015352d5b9387eaf5078e1ba5e0ea2
